src/components/utils/Crypto.js

```diff
@@ -1,104 +1,112 @@
-}
+  let mnemonic = await SecureStore.getItemAsync('mnemonic');
+  const wallet = new ethers.Wallet.fromMnemonic(mnemonic);
+  mnemonic = null;
+  return wallet.connect(provider);
+};
 
 const getBalance = async () => {
   const provider = new ethers.providers.InfuraProvider('kovan', env.infuraKey);
-  const walletAddress = await getWalletAddress()
-  const contractDai = new ethers.Contract(env.DAI.contractAddress, env.DAI.contractAbi, provider)
-  const daiBalanceinWei = await contractDai.balanceOf(walletAddress)
-  return daiBalanceinWei
-}
+  const walletAddress = await getWalletAddress();
+  const contractDai = new ethers.Contract(env.DAI.contractAddress, env.DAI.contractAbi, provider);
+  const daiBalanceinWei = await contractDai.balanceOf(walletAddress);
+  return daiBalanceinWei;
+};
 
 const signDAITransaction = async (amountInDai, toAddress) => {
-  const wallet = await getEthersWallet()
-  const numberOfTokensToSend = ethers.utils.parseUnits(amountInDai, Currencies.DAI.decimals)
-  const contract = new ethers.Contract(env.DAI.contractAddress, env.DAI.contractAbi, wallet)
+  const wallet = await getEthersWallet();
+  const numberOfTokensToSend = ethers.utils.parseUnits(amountInDai, Currencies.DAI.decimals);
+  const contract = new ethers.Contract(env.DAI.contractAddress, env.DAI.contractAbi, wallet);
 
-  var options = {
+  const options = {
     gasLimit: 200000,
     gasPrice: ethers.utils.parseUnits('10.0', 'gwei')
   };
 
   try {
-    const tx = await contract.transfer(toAddress, numberOfTokensToSend, options)
-    await tx.wait()
-    return tx
+    const tx = await contract.transfer(toAddress, numberOfTokensToSend, options);
+    await tx.wait();
+    return tx;
   } catch (error) {
-    throw error
+    throw error;
   }
-}
+};
 
 const signMessage = async (message) => {
-  const wallet = await getEthersWallet()
-  return wallet.signMessage(message)
-}
-
-const verifyMessage = async (message, signature) => {
-  return ethers.utils.verifyMessage(message, signature)
-}
-
-const weiToInteger = (amountInWei) => {
-  return formatToCurrency(Number(ethers.utils.formatEther(amountInWei)))
-}
-
-const formatToCurrency = (number) => {
-  return Number(number).toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 })
-}
+  const wallet = await getEthersWallet();
+  return wallet.signMessage(message);
+};
 
-export default { generateMnemonic, deriveWalletAddress, tryMnemonic, getWalletAddress, validateAddress, getEthersWallet, getStoredMnemonic, getBalance, signDAITransaction, weiToInteger, formatToCurrency, signMessage, verifyMessage }
+const verifyMessage = async (message, signature) => ethers.utils.verifyMessage(message, signature);
+
+const weiToInteger = (amountInWei) => formatToCurrency(Number(ethers.utils.formatEther(amountInWei)));
+
+const formatToCurrency = (number) => Number(number).toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 2 });
+
+export default {
```

Comment:
I think there's a better way of exporting these functions. Maybe something like

```export default {
  function1: () => {
    ...
  },
  function2..........
```

What do you think?

Reply:
In `JS` `exports` are usually always declared at the end of a file. This is because finding which `functions` are being exported is always in the same place on every file and not every function in a file is exported.